src/app/components/SubHeading/index.jsx

```diff
@@ -1,7 +1,18 @@
 import React from 'react';
+import styled from 'styled-components';
 import { extractText } from '../../helpers/blockHandlers';
 import { textBlockPropTypes } from '../../models/propTypes/text';
 import { textDefaultPropTypes } from '../../models/propTypes';
+import mediaQuery from '../../helpers/mediaQueries';
+
+const StyledSubHeading = styled.h2`
+  font-size: 1.25em;
+  line-height: 1.5em;
+  ${mediaQuery.desktopOnly} {
```

Comment:
This should be the `smartPhoneOnly` media query, with the values below added for 'desktopOnly'. 

```
 ${mediaQuery.desktopOnly} {
    font-size: 2em;
    line-height: 2.25em;
}
```